<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceEditorForm</class>
 <widget class="QDialog" name="SourceEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Source information</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="sourceTypeLabel">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>sourceTypeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="sourceTypeComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sourcePublicationLabel">
        <property name="text">
         <string>Publication</string>
        </property>
        <property name="buddy">
         <cstring>sourcePublicationEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="noteEditLabel">
        <property name="text">
         <string>Notes</string>
        </property>
        <property name="buddy">
         <cstring>noteEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout2" stretch="0,0">
        <item>
         <widget class="KRichTextEdit" name="noteEdit"/>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QPushButton" name="noteEditButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-entry"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sourceTitleLabel">
        <property name="text">
         <string>Title</string>
        </property>
        <property name="buddy">
         <cstring>sourcetTitleEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="sourcetTitleEdit">
        <property name="placeholderText">
         <string>Title of the source</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sourceAuthorLabel">
        <property name="text">
         <string>Author</string>
        </property>
        <property name="buddy">
         <cstring>sourceAuthorEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="sourceAuthorEdit">
        <property name="placeholderText">
         <string>Author or publisher of the source</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="sourceConfidenceLabel">
        <property name="text">
         <string>Confidence</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="sourceConfidenceCombobox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Parent</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="sourceParentDisplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>No parent selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sourceParentAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::ListAdd"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sourceParentPickButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::InsertLink"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="sourcePublicationEdit">
        <property name="placeholderText">
         <string>How or where the source is published</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>QTextEdit</extends>
   <header>KRichTextEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>SourceEditorForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>SourceEditorForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
